pydevices/HtsDevices/acq2106_423st.py

```diff
@@ -255,7 +255,7 @@ def run(self):
                         else:
                             self.full_buffers.put(buf)
 
-    def init(self):
+    def init(self, is_arm=1):
```

Comment:
Does `is_arm` mean it is already armed, or that we want to arm it?

Reply:
Maybe the name of the variable is confusing. What it means is that:

a) if the value is 1, then, the super-class will arm it, i.e. by starting the MDSWorker() thread.
b) if the value is 0. then, the super-class will not arm it. I will be armed by the transient device instead. Which in this case is the code acq2106_423tr.py.

I needed to make the distinction, otherwise, when we try to do a transient recording, the super-class will arm the ACQ before we can start the transient recording.

Reply:
Maybe it should be:

0: if we are not using a transient recording, and so it will be armed, as usual, by the super-class device.
1 : if we want to arm it when only starting a transient recording

Could this less confusing values for the variable?